indico/modules/events/abstracts/controllers/management.py

```diff
@@ -123,3 +142,79 @@ class RHAbstractListStaticURL(RHAbstractListBase):
 
     def _process(self):
         return jsonify(url=self.list_generator.generate_static_url())
+
+
+class RHCreateAbstract(RHAbstractListBase):
+    def _process(self):
+        abstract_form_class = make_abstract_form(self.event_new)
+        form = abstract_form_class(event=self.event_new)
+        if form.validate_on_submit():
+            data = form.data
+            if 'attachments' in data:
+                del data['attachments']
+            if not isinstance(data['submitted_for_tracks'], set):
+                submitted_for_track = set()
+                submitted_for_track.add(data['submitted_for_tracks'])
+                data['submitted_for_tracks'] = submitted_for_track
```

Comment:
You can replace those three lines with this:

    data['submitted_for_tracks'] = {data['submitted_for_tracks']}

Also, I'd rather check if you already got a `Track` object and put it in a set instead of the other way round (what if we ever changed it to use a list - it's unlikely but would anyone think of changing the `isinstance` check here in that case?)